src/components/NavigationSection.svelte

```diff
@@ -69,6 +76,22 @@
     }
   }
 
+  function getCategories (items) {
+    const categoriesOject = {}
+    items.forEach(i => {
+      if (!categoriesOject.hasOwnProperty(i.category)){
+        categoriesOject[i.category] = []
+      }
+      categoriesOject[i.category].push(i)
+    })
+    // since sveltejs doesn't allow to loop through objects
+    const categories = []
+    for (const category in categoriesOject) {
+      categories.push({title: category, items: categoriesOject[category]})
```

Comment:
Ok I think the case will never happen but ok, it is irrelevant to rename all the items.